### ISSUE TITLE
Input font should be adjustable separately

### ISSUE THREAD

**Issue by peterno3** [Chatterino/chatterino2#2335]
The input font should be adjustable separately from the standard message display font. On my PC, the input font appears to be 4x the size of the message font. I don't have a way to adjust it without adjusting the size of all text

**Comment by fourtf** [comment]
Could you provide a screenshot of the input box?

**Comment by peterno3** [comment]
Of course, it was happening consistently before I posted, but as soon as i went to take a screenshot it resolved itself. I think it must have been related to monitor scaling or something. Everything looks ok now though, thanks.

**Comment by CaptainFlint** [comment]
Yes, there is such problem, I have it too. It's is indeed related to scaling. See the two screenshots: first is taken with 100% scaling, the second one is with 150% scaling. You can see that in the first one the input box font size is very close to that in the chat window; and in the second screenshot it's much larger, unproportionally so.

![chatterino100](https://user-images.githubusercontent.com/6475044/103571386-d697a980-4edb-11eb-84c0-42eb0bbef6b7.png)
![chatterino150](https://user-images.githubusercontent.com/6475044/103571390-d7304000-4edb-11eb-99d1-c2bb40e5879c.png)

**Comment by peterno3** [comment]
Reopening because after rebooting my PC, the issue came back. Below is a screenshot

![image](https://user-images.githubusercontent.com/5396987/103601242-194a8900-4ebe-11eb-8adc-28154113fb7c.png)

After closing and reopening the program a few times the issue seems to go away

![image](https://user-images.githubusercontent.com/5396987/103601403-6d556d80-4ebe-11eb-9312-3375bdafee58.png)

I have Chatterino on a monitor with only 100% scaling (2560 x 1440 resolution), but i have a second monitor that's at 250% scaling (3840 x 2160 resolution) which is possibly messing with it

**Comment by CaptainFlint** [comment]
In my case closing and reopening did not fix it, but I was testing it in a single-monitor environment.
